### DISCUSSION TITLE
so-steno-start: command not found

### DISCUSSION THREAD

**Discussion by ekky19** [Security-Onion-Solutions/securityonion#8147] [Unsupported Versions] [answered]
I installed SO Version: 2.3.130 in import mode, and couldn't start the steno. Also when I import a pcap file, I can see the logs in dashboard, hunt, and zeek but not in alerts. I have done 3 different installations and it is always the same.

[ekrem@importt stenographer]$ **sudo so-steno-start**
sudo: so-steno-start: command not found

I have also tried  **sudo so-pcap-start** and  **sudo so-pcap-restart** and the result is still the same.

this is the hardware info:
![image](https://user-images.githubusercontent.com/6872571/174422245-a7beb6ca-c380-4518-b3e9-77b7d6282b01.png)

##########################################################################
here is the log from **stenographer.log**

ABORTABORTABORT
/usr/bin/stenotype() [0x4081f8]
/usr/bin/stenotype() [0x42e4a4]
/lib64/libc.so.6(__libc_start_main+0xf5) [0x7f6bf9ec5555]
/usr/bin/stenotype() [0x40412b]
2022/06/18 03:49:43 Stenotype stopped after 45.638021093s: stenotype wait failed: signal: aborted
2022/06/18 03:49:43 Stenotype ran for too little time, crashing to avoid stenotype crash loop
2022-06-18T03:52:14.435043Z T:8c95b7 [stenotype.cc:567] Starting, page size is 4096
2022-06-18T03:52:14.435183Z T:8c95b7 [stenotype.cc:594] Setting up AF_PACKET sockets for packet reading
2022-06-18T03:52:22.832341Z T:8c95b7 [stenotype.cc:619] CHECK(SUCCEEDED(__check_success_error__)) builder.Bind(flag_iface, &v3): No such device

----------
          ID: stenoca
    Function: file.directory
        Name: /opt/so/conf/steno/certs
      Result: True
     Comment: The directory /opt/so/conf/steno/certs is in the correct state
     Started: 13:53:35.037426
    Duration: 2.668 ms
     Changes:
----------
          ID: pcapdir
    Function: file.directory
        Name: /nsm/pcap
      Result: True
     Comment: The directory /nsm/pcap is in the correct state
     Started: 13:53:35.040550
    Duration: 2.375 ms
     Changes:
----------
          
          ID: so-steno
    Function: docker_container.running
      Result: True
     Comment: Created container 'so-steno'
     Started: 13:53:35.147759
    Duration: 1446.214 ms
     Changes:
              ----------
              container_id:
                  ----------
                  added:
                      2fcde6839da94eab0f584892284df08acd71c1de2ba74bbd07e2cc4b8fdfe55f
              state:
                  ----------
                  new:
                      running
                  old:
                      None

**Comment by xfaith** [comment, marked as answer]
For the first question:  There is not a command so-steno-start .  
The command to start/stop steno would be so-pcap-start (stop)(restart).
---I saw another post with the same errors on the discussion forums 

--Doug asked for the following: 
What are the hardware specs of your machine?
How many NICs do you have?
What kind of NICs are they?
What options did you choose during Setup?

**One thing I noticed is that you only have 1 NIC displayed in your VM image, you should have a min of 2. (1 for Management/1 for Sniffing)  

2nd  Question: Sounds like Suricata is not alerting.  Have you verified your HOME_NET?

**Reply by ekky19** [reply]
It is import node…I guess you misses that part.

**Reply by xfaith** [reply]
Yes, sorry: 

But from the docs:  
![image](https://user-images.githubusercontent.com/1995708/174425051-9d0c6902-d5da-45a0-a321-61d8895ac898.png)

**Reply by ekky19** [reply]
I see mate… thank you.
